e2e/test/scenarios/visualizations-tabular/table.cy.spec.js

```diff
@@ -489,6 +489,25 @@ describe("scenarios > visualizations > table", () => {
     cy.tick(5000);
     cy.findByTestId("query-builder-main").findByText("Waiting for results...");
   });
+
+  it("should support 'Local symbol' in 'Currency label style' viz setting", () => {
```

Comment:
This may be the first e2e test for "Currency label style" setting, I couldn't find any :see_no_evil: